src/box/memtx_tx.c

```diff
@@ -1904,6 +1912,20 @@ memtx_tx_track_gap_slow(struct txn *txn, struct space *space, struct index *inde
 	return 0;
 }
 
+/**
```

Comment:
It's enough to have one comment in a header file, don't need to duplicate it in a source file

Reply:
Thanks, fixed.

Reply:
Fixed